blt/blt.yml

```diff
@@ -28,8 +28,6 @@ disable-targets:
     toggle:
       modules: true
   tests:
-    frontend:
```

Comment:
Just curious, what does removing `frontend` from the tests do?

Reply:
We had previously disabled that command because we weren't using it. Removing it from `disable-targets` re-enables it and we're using it execute the percy commands. https://docs.acquia.com/blt/extending-blt/#disabling-a-command